modules/render.js

```diff
@@ -0,0 +1,124 @@
+import Todo from './todo.js';
+import updateStatus from './todo-status.js';
+
+const todosList = document.querySelector('.todos-list');
+const todoInput = document.querySelector('.add-todo');
+const addBtn = document.querySelector('.add');
+const clearCompletedBtn = document.querySelector('.clear');
+let todosArr = [];
+
+const updateIndex = (index) => {
+  for (let i = index + 1; i < todosArr.length; i += 1) {
+    todosArr[i].index -= 1;
+  }
+};
+
+const updateCheckBox = (index, bool) => {
+  updateStatus(index, bool, todosArr);
+};
+
+const render = () => {
+  todosList.innerHTML = null;
+  const todo = new Todo();
+  if (localStorage.getItem('todos')) {
+    todosArr = JSON.parse(localStorage.getItem('todos'));
+  }
+
+  const removeTodo = (arrIndex) => {
+    updateIndex(arrIndex);
+    todo.remove(arrIndex, todosArr);
+  };
+
+  const editDescription = (index, ele) => {
+    todosArr[index].description = ele.value;
+    todo.update(todosArr);
+    render();
+  };
+
+  for (let i = 0; i < todosArr.length; i += 1) {
+    const todoDiv = document.createElement('div');
```

Comment:
OPTIONAL: I think it's a good idea to use the innerHTML property and write this block of code using HTML syntax instead of creating the elements one by one. It could save a lot of time when you are coding.